### PR TITLE
Fixes non-portable path to file "platform/Platform.hpp"

### PR DESCRIPTION
**Pull request by ANS-UXI**
Fixes: #4616

Refactored `#include "platform/platform.hpp"` to `#include "platform/Platform.hpp"` in Framework.cpp and DownloadResourcesLegacyActivity.cpp file.

### CODE REVIEW

**Comment by biodranik** [comment]
There is also `platform.hpp` file in the root `platform` folder. Did you recompile everything and tested that your fix works?

**Comment by ANS-UXI** [comment]
> There is also `platform.hpp` file in the root `platform` folder. Did you recompile everything and tested that your fix works?

Can you please guide me on this recompiling path like how to do it?

**Comment by ANS-UXI** [comment]
> There is also `platform.hpp` file in the root `platform` folder. Did you recompile everything and tested that your fix works?

Now I tested it and I guess it works!
image

**Comment by ANS-UXI** [comment]
Also, the warning is gone!

**Comment by ANS-UXI** [comment]
Thanks for informing @rtsisyk , I have made the changes and rebuild the project successfully, please have a look!

File: android/jni/app/organicmaps/isolines/IsolinesManager.cpp [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #include <jni.h>
 #include "app/organicmaps/Framework.hpp"
 #include "app/organicmaps/core/jni_helper.hpp"
-#include "app/organicmaps/platform/Platform.hpp"
+#include "app/organicmaps/platform/PlatformNew.hpp"
```
Comment (biodranik) [inline android/jni/app/organicmaps/isolines/IsolinesManager.cpp:4]: Let's rename it to AndroidPlatform instead. "New" is confusing here.

**Review by biodranik — CHANGES_REQUESTED** [review]

File: android/jni/app/organicmaps/isolines/IsolinesManager.cpp (hunk 1 above)
Comment (ANS-UXI) [reply android/jni/app/organicmaps/isolines/IsolinesManager.cpp]: It's done!

**Review by biodranik — APPROVED** [review]
Thank you for your contribution!

**Comment by ANS-UXI** [comment]
> Thank you for your contribution!

You're welcome! It's always fun :D

**Pull request merged**